src/scripts/build.sh

```diff
@@ -66,6 +66,9 @@ fi
 # The context must be the last argument.
 build_args+=("$PARAM_DOCKER_CONTEXT")
 
+# recreate build_args array so it doesn't group multiple args as a single variable
+read -a build_args < <( echo "${build_args[@]}" )
```

Comment:
I'm thinking that a better strategy will be to explicitly set the IFS to space and replace `[@]` with `[*]`:

```
# The context must be the last argument.
build_args+=("$PARAM_DOCKER_CONTEXT")

old_ifs="$IFS"
IFS=' '

set -x
docker build ${build_args[*]}
set +x

IFS="$old_ifs"
```

Seems like it does the trick: https://app.circleci.com/pipelines/github/CircleCI-Public/docker-orb/790/workflows/c8f9b117-bfd4-45e3-a030-82613adfbbd6/jobs/13502?invite=true#step-109-78

Reply:
Thoughts?

Reply:
Ahh good idea. 

That works, updating now